lua/lsp/null-ls/formatters.lua

```diff
@@ -63,7 +63,7 @@ function M.list_configured(formatter_configs)
 end
 
 function M.setup(filetype, options)
-  if formatters_by_ft[filetype] and not options.force_reload then
+  if (formatters_by_ft[filetype] and not options.force_reload) or not lvim.lang[filetype] then
```

Comment:
Can you reverse the condition's order please :)

Reply:
sure